assets/schema_input.json

```diff
@@ -6,7 +6,7 @@
     "type": "array",
     "items": {
         "type": "object",
-        "required": ["sample", "design", "fastq_1"],
+        "required": ["sample", "design", "fastq_1", "panel"],
```

Comment:
Is panel a required option?

Reply:
I don't think panel should be required, since we should be able to accept either `panel` or `panel_file` as inputs. This is documented here in [usage.md](https://github.com/nf-core/pixelator/blob/42424bb2977d73babba67a6766f2a1c8334b958b/docs/usage.md?plain=1#L44).

Reply:
Allright @milenaabernardi can you remove that as a required option?